api/app/main.py

```diff
@@ -412,3 +413,10 @@ def post_raster_geotiff(raster_geotiff: RasterGeotiffModel):
     return JSONResponse(
         content={"download_url": presigned_download_url}, status_code=200
     )
+
+
+@app.get("/google-floods-gauges")
```

Comment:
If we're gonna add a bunch of integrations, it will be worthwile to think about a standardized approach and common architecture. And maybe expose them through a sub endpoint of the API to keep things cleaner. wdyt @lowriech ?

Reply:
I think that's a good question, @ericboucher. Do you see other APIs we're already leveraging fitting into this integration category (kobo, hdc stats, others)? Or are you thinking just for new integrations? 

I'm wary of premature abstraction and trying to determine if I think now is the right time to align on this standardized approach.

Reply:
It seems like low-hanging fruit to abstract once we decide on a pattern.  So yes, agree we should think about it, but not sure about the velocity of adding entirely new providers.  Would vote to wait until we see more pattern around this.